tlsr9/crypto/mbedtls/internal/ecp_alt_b9x_backend.c

```diff
@@ -263,6 +263,25 @@ static mont_curve_t x25519 = {
 };
 #endif
 
+#if defined CONFIG_SOC_RISCV_TELINK_B93
```

Comment:
are there any difference compare to CONFIG_SOC_RISCV_TELINK_B92?

Reply:
removed duplicated code, please recheck